Sources/PointFree/TODO.swift

```diff
@@ -463,3 +477,73 @@ public func require2<A, B, Z>(_ x: T3<A, B?, Z>) -> T3<A, B, Z>? {
 public func lower<A>(_ tuple: Tuple1<A>) -> A {
   return get1(tuple)
 }
+
+import Cryptor
+
+extension PartialIso where A == String, B == String {
+  public static func decrypted(withSecret secret: String) -> PartialIso<String, String> {
+    return PartialIso(
+      apply: { PointFree.decrypted(text: $0, secret: secret) },
+      unapply: { PointFree.encrypted(text: $0, secret: secret) }
+    )
+  }
+
+  public static var appDecrypted: PartialIso<String, String> {
+    return .decrypted(withSecret: AppEnvironment.current.envVars.appSecret)
+  }
+}
+
+public func encrypted(text plainText: String, secret: String) -> String? {
```

Comment:
can remove this once I update swift-web. just didnt want to mix in those changes yet